### ISSUE TITLE
Are Hg19 or other genome versions/species supported?

### ISSUE THREAD

**Issue by vschulz** [ciceklab/ECOLE#1]
Are Hg19 or other genome versions/species supported?  I see that HG38 was used in the paper, but it was not clear if other genome versions would work.  If so are any changes needed for either initial run or fine tuning?  Are the values in ecole_stats.txt usable for different genome versions or different exome capture methods?
Thanks,
Vince

**Comment by alperylmzz** [comment]
The ECOLE model was trained with WES reads aligned to the HG38 reference genome. Unfortunately, other genome versions are not supported for fine-tuning. You should convert your BAM file to HG38.
 
The values in ecole_stats.txt contain the mean and standard deviation statistics of read depth and are used to normalize the data. Using different capture methods can affect the performance. Therefore, we suggest computing the mean and standard deviation values of your entire dataset.
